docs/contributing/commit-signing.mdx

```diff
@@ -101,6 +101,17 @@ Refer to
   your terminal. Refer to
   [this troubleshooting document](https://gist.github.com/paolocarrasco/18ca8fe6e63490ae1be23e84a7039374)
   for more help with this error
+- If you have already committed your changes and forgot to configure gpg signing you can fix it by
+  doing a rebase like the following targeting the hash of the previous commit to the one you did (it
+  will sign all commits from there):
+
+```
+git rebase --exec 'git commit --amend --no-edit -n -S' -i <previous-commmit-hash>
+```
+
+Beware that if you had already pushed your changes to origin, you are going to have to do a force
+push in git after applying the fix which shouldn't be harmful on side branches but one needs to pay
+attention just in case.
```

Comment:
```suggestion
  ```
  git rebase --exec 'git commit --amend --no-edit -n -S' -i <previous-commmit-hash>
  ```

  Beware that if you had already pushed your changes to origin, you are going to have to do a force
  push in git after applying the fix which shouldn't be harmful on side branches but one needs to pay
  attention just in case.
```